Admin/Extension/MenuNodeReferrersExtension.php

```diff
@@ -21,20 +21,47 @@
  */
 class MenuNodeReferrersExtension extends AdminExtension
 {
+    /**
+     * @var string
+     */
+    private $formTab;
+
+    /**
+     * @var string
+     */
+    private $formGroup;
+
+    public function __construct($formGroup = 'form.group_menu', $formTab = 'form.tab_menu')
+    {
+        $this->formGroup = $formGroup;
+        $this->formTab = $formTab;
+    }
+
     public function configureFormFields(FormMapper $formMapper)
     {
+        if ($formMapper->hasOpenTab()) {
+            $formMapper->end();
+        }
```

Comment:
imho this would justiy a 2.0.1 release and similar, it looks so unusable in the admin.